src/Controls/tests/UITests/Tests/Issues/Issue18946.cs

```diff
@@ -0,0 +1,25 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.AppiumTests.Issues
+{
+	public class Issue18946 : _IssuesUITest
+	{
+		public override string Issue => "[iOS] Shell Toolbar items not displayed";
+
+		public Issue18946(TestDevice device) : base(device)
+		{}
+
+		[Test]
+		public void ToolbarItemsShouldBeVisible()
+		{
+			this.IgnoreIfPlatforms(new TestDevice[] { TestDevice.Android, TestDevice.Mac, TestDevice.Windows });
```

Comment:
Could we enabled the tests for the test of platforms?

Reply:
Sure. Done!